blazor/accumulation-chart/legend.md

```diff
@@ -293,6 +293,47 @@ When the legend items exceed legend bounds, paging will be enabled by default. E
 
 ![Paging for Legend in Blazor Accumulation Chart](images/legend/blazor-accumulation-chart-legend-paging.png)
 
+### Legend Paging Customization
+
+The Legend Page number can be cutomized by setting arrow size , text size and color.
```

Comment:
add paging number can be customized textstyle and arrow should be customized arrow size API. explain clearly also API name `ArrowSize`

Reply:
Description changed